client/hotrod-client/src/test/java/org/infinispan/client/hotrod/MillisecondExpirationTest.java

```diff
@@ -0,0 +1,32 @@
+package org.infinispan.client.hotrod;
+
+import java.util.concurrent.TimeUnit;
+
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.manager.EmbeddedCacheManager;
+import static org.infinispan.server.hotrod.test.HotRodTestingUtil.hotRodCacheConfiguration;
+import org.infinispan.test.fwk.TestCacheManagerFactory;
+import static org.testng.AssertJUnit.assertFalse;
+import org.testng.annotations.Test;
+
+/**
+ * @author gustavonalle
+ * @since 7.2
+ */
+@Test(testName = "client.hotrod.MillisecondExpirationTest", groups = "functional")
+public class MillisecondExpirationTest extends DefaultExpirationTest {
+
+   @Override
+   protected EmbeddedCacheManager createCacheManager() throws Exception {
+      ConfigurationBuilder builder = hotRodCacheConfiguration(
+            getDefaultStandaloneCacheConfig(false));
+      return TestCacheManagerFactory.createCacheManager(builder);
+   }
+
+   @Test
+   public void testDefaultExpiration() throws Exception {
+      remoteCache.put("Key", "Value", 50, TimeUnit.MILLISECONDS);
+      Thread.sleep(100);
```

Comment:
hmm, was looking at ```org.infinispan.client.hotrod.ExpiryTest``` and it seems it always passes since all ```cache.get``` returns null, regardless of expirations. Missing setting the correct ```keyEquivalence``` maybe?